dev/TeachingTip/TeachingTip.cpp

```diff
@@ -25,6 +26,11 @@ winrt::AutomationPeer TeachingTip::OnCreateAutomationPeer()
     return winrt::make<TeachingTipAutomationPeer>(*this);
 }
 
+void TeachingTip::RemovePopupOnUnloadEvent(IInspectable const& /* sender */, winrt::RoutedEventArgs e)
```

Comment:
Looks like both arguments are unused, so leave it commented out?
```suggestion
void TeachingTip::RemovePopupOnUnloadEvent(winrt::IInspectable const& /* sender */, winrt::RoutedEventArgs const& /* e */)
```

Reply:
(You'll need to commit this suggestion too before it will compile)